internal/contour/listener_test.go

```diff
@@ -37,7 +37,7 @@ import (
 func TestListenerCacheContents(t *testing.T) {
 	tests := map[string]struct {
 		contents map[string]*v2.Listener
-		want     []proto.Message
+		want     []cachetypes.Resource
```

Comment:
If we make a `xds` package that just contained type aliases this would read much more clearly as `xds.Resource`.